langchain/src/prompts/tests/chat.test.ts

```diff
@@ -0,0 +1,157 @@
+import { expect, test } from "@jest/globals";
+import { ChatPromptTemplate } from "../chat.js";
+import { PromptTemplate } from "../prompt.js";
+
+test("Test format", async () => {
+  const systemPrompt = new PromptTemplate({
```

Comment:
Can we make this test slightly more comprehensive by having 1 variable shared by the 2 prompts

Reply:
will do